ext/ReactiveMPProjectionExt/rules/out.jl

```diff
@@ -1,19 +1,33 @@
+# First method: when there's no projection form
+function create_project_to(::CVIProjection{R, S, Nothing}, q_out::Any, q_out_samples::Any) where {R, S}
+    T = ExponentialFamily.exponential_family_typetag(q_out)
+    q_out_ef    = convert(ExponentialFamilyDistribution, q_out)
+    conditioner = getconditioner(q_out_ef)
+    return ProjectedTo(T, size(first(q_out_samples))...; conditioner = conditioner, parameters = ExponentialFamilyProjection.DefaultProjectionParameters())
+end
+
+function create_project_to(method::CVIProjection{R, S, OF}, ::Any, ::Any) where {R, S, OF <: ProjectedTo}
+    return method.out_prjparams 
+end
+
+function create_project_to(method::CVIProjection{R, S, OF}, ::Any, ::Any) where {R, S, OF <: ProjectionParameters}
```

Comment:
Here as well, variables haven't been defined

Reply:
Yes I see sorry, I wonder why it does not fails in the tests